WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -2167,7 +2167,52 @@ static GSourceFuncs _handlerIntervention =
                 }
                 g_mkdir_with_parents(wpeDiskCachePath, 0700);
 
+#if WEBKIT_CHECK_VERSION(2, 38, 0)
```

Comment:
Let's not use the version here but I propose to make a separate build flag that distinguishes "old environment way" vs "new direct way" (think of a good name for this :) ). That way it is more clear what the intention is and if needed we could switch in depend of the version